site/docs/tasks/using-opa.md

```diff
@@ -15,7 +15,8 @@ package dataapi.authz
 rule [{}] { true }
 ```
 
-You can also add conditions like
+To add a condition a new rule named `rule` should be added as shown below.
```

Comment:
This statement is misleading. It appears in the Default policies section. Should be in some other "general' section - an explanation about syntax - only `rule` is allowed, the output is either an empty object (representing allow action) or an object including action structure defined in the taxonomy as well as the policy description.

Reply:
OK thanks will add a new section.